### PR TITLE
Update turn-on-or-off-offline-messages-for-admins.md

### PR DESCRIPTION
**Pull request by pupara**
Added Notes
Replaced SkypeOnlineConnector with MicrosoftTeams

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-offline-messages-for-admins.md [hunk 1]
```diff
@@ -59,12 +59,13 @@ If you need to know more, see [Connect to all Office 365 services in a single Wi
 
 2. In the **Windows PowerShell** window, connect to your Microsoft 365 or Office 365 by running:
 
-    > [!NOTE]
-    > You only have to run the **Import-Module** command the first time you use the Skype for Business Online Windows PowerShell module.
+> [!NOTE]
+> Skype for Business Online Connector is currently part of latest Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-offline-messages-for-admins.md:63]:
```suggestion
> The Skype for Business Online Connector is currently part of the latest Teams PowerShell module.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by pupara** [comment]
@JohanFreelancer9 , i am still reviewing these pull request and that why these are still in draft.

**Comment by JohanFreelancer9** [comment]
@pupara I thought so, thank you. I'll do a final copy review when you're done.

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**